### ISSUE TITLE
[FR] Specify if a particular part can exist in "non integer" quantities

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#2778]
It only makes sense to allow non-integer quantities for certain types of parts.

For example, a microcontroller cannot exist in a quantity of 0.35

We should add a boolean field to the Part model to indicate whether a part is allowed to exist in a non-integer quantity. Then, some business logic would have to be created to reflect this idea.

**Comment by SchrodingersGat** [comment]
Not stale

**Comment by SchrodingersGat** [comment]
Not stale

**Comment by SchrodingersGat** [comment]
Not stale, adding to "horizon"

**Comment by matmair** [comment]
@SchrodingersGat does this still have relevance?

**Comment by SchrodingersGat** [comment]
Yes, low priority but I would still like to get to it at some stage.

**Comment by SchrodingersGat** [comment]
I may offload it to a custom plugin
